app/controllers/comments_controller.rb

```diff
@@ -3,12 +3,18 @@
 # Controller Comments
 class CommentsController < ApplicationController
   skip_before_action :verify_authenticity_token
+  before_action :user_fix_comment?, only: %i[update destroy]
   def create
-    @micropost = Micropost.find_by(id: params[:micropost_id])
-    @comment = @micropost.comments.build(content_comment: params[:micropost][:content_comment],
-                                         parent_comment_id: params[:parent_comment_id])
-    @comment.user = current_user
-
+    id = params[:micropost][:micropost_id]
+    if id.present?
+      @micropost = Micropost.find_by(id: id)
+      @comment = @micropost.comments.build(comment_params)
+      @comment.image.attach(params[:micropost][:image])
```

Comment:
With ActiveStorage, we don't need attach image.
`@comment = @micropost.comments.build(comment_params)` is enought